Library/Formula/svtplay-dl.rb

```diff
@@ -22,6 +15,7 @@ class SvtplayDl < Formula
   end
 
   depends_on "rtmpdump"
+  depends_on "python"
```

Comment:
I think a better solution is to install the requests[security] dependencies, which will use PyOpenSSL with pyca/cryptography in preference to the built-in ssl module, which is not as flexible. Let me know if you need help enumerating those dependencies. Requests will automatically use them if they are available.